config/charts/bbr/README.md

```diff
@@ -0,0 +1,39 @@
+# Body-based routing
+
+A chart to the body-based routing deployment and service.
+
+
+## Install
+
+To install a body-based router named `body-based-router`, you can run the following command:
+
+```txt
+$ helm install body-based-router ./config/charts/bbr
```

Comment:
Can we clarify if one needs to install this per what exactly?

Reply:
No, I meant to clarify if one should install this per pool, per gateway or per cluster

Reply:
Done